src/main/java/org/folio/search/service/config/BrowseConfigService.java

```diff
@@ -80,7 +81,9 @@ public void deleteTypeIdsFromConfigs(@NonNull BrowseType type, @NonNull List<Str
     }
     var configs = repository.findByConfigId_BrowseType(type.getValue());
     for (BrowseConfigEntity config : configs) {
-      var newTypeIds = new ArrayList<>(config.getTypeIds());
+      var newTypeIds = Optional.ofNullable(config.getTypeIds())
+        .map(LinkedList::new)
```

Comment:
Why linked list?

Reply:
Didn't see any operations by index so replaced it, now I see it uses them internally in iterator, reverting